core/src/main/resources/error/error-classes.json

```diff
@@ -5569,5 +5569,15 @@
     "message" : [
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
+  },
+  "_LEGACY_ERROR_TEMP_3043" : {
+    "message" : [
+      "User exception: <msg>"
+    ]
+  },
+  "_LEGACY_ERROR_TEMP_3044" : {
```

Comment:
Shall we define a proper error class? There is `NOT_NULL_CONSTRAINT_VIOLATION` that seems relevant. It is currently used to validate NOT NULL constraints for array elements and map values.